ocaml/idl/datamodel.ml

```diff
@@ -9019,7 +9019,7 @@ let pgpu =
         {
           param_type = (Ref _vgpu_type);
           param_name = "value";
-          param_doc = "The VGPU type to enable";
+          param_doc = "The VGPU types to enable";
```

Comment:
Thanks. @kc284 
Will change it. So I think the line below is wrong and should be types-->type.
Would you please help to confirm? Thanks.
https://github.com/minli1/xen-api/blob/48a41e08b174fb4e65ef833f8057715f3f9f1bf3/ocaml/idl/datamodel.ml#L9066